src/components/ha-sidebar.html

```diff
@@ -93,6 +93,11 @@
       .dev-tools {
         padding: 0 8px;
         opacity: var(--dark-secondary-opacity);
+        display:var(--display-dev)!important;
```

Comment:
Make a new class for this. You are applying padding and opacity to a different element now.

Reply:
That is not the case. I am doing that on purpose. I am trying to hide two sensitive elements with one variable.
<img width="201" alt="config" src="https://user-images.githubusercontent.com/8059190/35197790-3bc78784-fee5-11e7-9ede-04cf2e24a9a2.png">

Reply:
Having a single variable is fine, but .dev-tools class brings a padding and your change applies this padding to a different element than before.

Reply:
Got it.